src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -50,8 +50,15 @@ public static class TreeExtensions
         /// </summary>
         /// <param name="catalog">The <see cref="RegressionCatalog"/>.</param>
         /// <param name="options">Trainer options.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[FastTreeRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastTreeWithOptions.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
         public static FastTreeRegressionTrainer FastTree(this RegressionCatalog.RegressionTrainers catalog,
-            FastTreeRegressionTrainer.Options options)
+        FastTreeRegressionTrainer.Options options)
```

Comment:
>         [](start = 0, length = 8)

nit: one more tab???

Reply:
yes

---
In reply to: [267035106](https://github.com/dotnet/machinelearning/pull/2999#discussion_r267035106) [](ancestors = 267035106)